packages/catalog-process/src/utilities/versionGenerator.ts

```diff
@@ -14,7 +14,7 @@ export const nextDescriptorVersion = (eservice: EService): string => {
     }
 
     return currentVersionNumber.data > max ? currentVersionNumber.data : max;
-  }, 1);
+  }, 0);
```

Comment:
The `0` accumulator in the reducer is needed in case the array has no descriptors.